app/javascript/components/event/EventEdit.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import Axios from 'axios';
+
+export default class EventEdit extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state =  {};
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleCancel = this.handleCancel.bind(this);
+  }
+
+  componentDidMount() {
+    Axios.get(`api/events/${this.props.match.params.id}`)
+      .then(res => {
+        this.setState(res.data);
+      })
+      .catch(error => console.log('error', error))
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    Axios.patch(`api/events/${this.props.match.params.id}`, this.state)
+      .then(response => console.log(response))
+      .then(data => {
+        this.props.history.push(`/events/${this.state.id}`);
+      })
+      .catch(error => console.log('error', error));
+  }
+
+  handleChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  handleCancel = () => {
+    this.props.history.push(`/events/${this.state.id}`);
+  }
+
+  render() {
+    const isStatus = this.state.status;
```

Comment:
why it has prefix 'is'? I don't think it's a good name. status or eventStatus are ok.